src/components/CompanyAndJobTitle/Sorter.js

```diff
@@ -1,22 +1,53 @@
-import React, { useCallback, useState } from 'react';
+import React, { useCallback } from 'react';
+import { useHistory } from 'react-router';
+import { useQuery } from 'hooks/routing';
+import qs from 'qs';
+
 import Select from 'common/form/Select';
 
+export const SORT_BY = {
+  LATEST_FIRST: 'LATEST_FIRST',
+  FEATURED_FIRST: 'FEATURED_FIRST',
+};
+
+export const sortByFromQuerySelector = query =>
+  query.sort_by || SORT_BY.LATEST_FIRST;
```

Comment:
can the default one to be `FEATURED_FIRST` ?

Reply:
Updated